ONSdigital/eq-author-app
# Fixed multple DOM elements for dashboard delete modal and added title…

### What is the context of this PR?
Multiple DOM elements on page for delete confirm modal. Also, questionnaire title was missing from delete confirm modal.

### How to review 
1. In Master, go to dashboard with multiple questionnaires listed.
2. View DOM and observe multiple empty delete confirm DOM elements attached to bottom of body
3. Click to delete a questionnaire, observer no title being displayed in modal
4. Repeat in branch and see no DOM elements attached (modal now only attached when clicking delete).
5. Click delete button and observe modal with questionnaire title
